iep-service/src/main/java/com/netflix/iep/service/AbstractService.java

```diff
@@ -24,7 +24,7 @@
  */
 public abstract class AbstractService implements Service {
 
-  private final String name;
```

Comment:
There is no need to remove the final from this usage here. You want to remove final from the name() method so it can be overriden by WebServer.

https://github.com/copperlight/iep/blob/a9ab01dec2897d23b86694ad0ea6166346d96104/iep-service/src/main/java/com/netflix/iep/service/AbstractService.java#L39